modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusService.java

```diff
@@ -56,7 +60,12 @@ public static void signalEvent(String jsonEvent) {
 
         Message replyMessage = Message.obtain(null, 0, 0, 0, null);
         replyMessage.setData(replyData);
-        sendReply(applicationMessenger, replyMessage);
+	try {
+	    applicationMessengerIsSet.await();
```

Comment:
We probably want to set an explicit timeout here.

Reply:
let's check if it helps with running autotests, then revisit